lib/pf/file_paths.pm

```diff
@@ -112,6 +114,7 @@ $log_conf_dir  = catdir( $conf_dir,"log.conf.d" );
 $generated_conf_dir   = catdir( $var_dir,"conf");
 $tt_compile_cache_dir = catdir( $var_dir,"tt_compile_cache");
 $pfconfig_cache_dir = catdir( $var_dir,"cache/pfconfig");
+$domains_chroot_dir = catdir( "/chroots");
```

Comment:
No kidding, this is exactly why I'm leaving it there. Having it part of the pf directory means that when doing a full backup of that directory, we'd be backing up all the chroots (that have mount points to a lot of places in the filesystem)